### PR TITLE
Sorting by field in all currencies template

### PR DESCRIPTION
**Pull request by fructusmortus**
Realized with js
image

### CODE REVIEW

File: static/js/sorttable.js [hunk 1]
```diff
@@ -0,0 +1,495 @@
+/*
```
Comment (nargiza-sarkulova) [inline static/js/sorttable.js]: Usually the minified versions of JS scripts is uploaded to decrease load times.

**Comment by nargiza-sarkulova** [comment]
> Realized with js
> image

Why there's such a large padding between rows? Could you include a screenshot showing the whole page?

**Comment by fructusmortus** [comment]
> 
> 
> > Realized with js
> > image
> 
> Why there's such a large padding between rows? Could you include a screenshot showing the whole page?

Images are the reason. 
image

**Review by nargiza-sarkulova — APPROVED** [review]
Please replace with minified JS plugin. After that 👍

**Pull request merged**